### ISSUE TITLE
Footnotes raise error in the console and don't work

### ISSUE THREAD

**Issue by feliperaul** [SublimeText-Markdown/MarkdownEditing#66]
This is what happens:

```Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 445, in is_enabled_
    raise ValueError("is_enabled must return a bool", self)
ValueError: ('is_enabled must return a bool', <MarkdownEditing.footnotes.MagicFootnotesCommand object at 0x1089c5390>)```

**Comment by sgroth** [comment]
Pushing this up, as this is still not working in the latest builds.

**Comment by michaelpacer** [comment]
It would be nice to link to this issue somewhere in the README (perhaps next to the footnote related keyboard shortcuts), because I've just spent the better part of a day trying to figure out how to change the default keybindings so as to make them work (they were conflicting with other shortcuts on my computer) and had no idea that the problem was one with the footnote scripts and not an error in what I was doing in messing around with shortcuts. 

I could have taken the time to actually try to understand the problem (and may have made some progress!) if I'd known from reading the README that there was an underlying issue that needed dealing with. Alas…

**Comment by felixhao28** [comment]
This issue is caused by using ST2 APIs. I have ported these footnotes related commands to ST3 (see commit 0e9da304eb390e3777dfd229dfd70a41fd98e319). The issue will be fixed v2.1.1. Thanks for all the information.
